.jscsrc

```diff
@@ -1,5 +1,6 @@
 {
 	"preset": "jquery",
 	"maximumLineLength": null,
-	"requireCamelCaseOrUpperCaseIdentifiers": null
+	"requireCamelCaseOrUpperCaseIdentifiers": null,
+	"requireCapitalizedComments": false
```

Comment:
Just tested them, and it seems all tests have passed :grin: 
So I think I'll uncomment these test for now.